packages/frontend/app/helpers/year-next.js

```diff
@@ -0,0 +1,7 @@
+import ENV from 'frontend/config/environment';
+
+export default function yearNext(year) {
```

Comment:
I like the helper creation, but looking at where this might be used I don't think it's worth the overhead since we can use a math helper in the template as easily. It's possible that a class based helper could take the course and output the year string *after* looking in the config to determine which output. That would be nice to consolidate. I'm not sure helpers can access services though. Worth investigating.